dpnp/backend/extensions/lapack/gesv.cpp

```diff
@@ -69,12 +64,25 @@ static sycl::event gesv_impl(sycl::queue &exec_q,
     const std::int64_t lda = std::max<size_t>(1UL, n);
     const std::int64_t ldb = std::max<size_t>(1UL, n);
 
-    const std::int64_t scratchpad_size =
-        mkl_lapack::gesv_scratchpad_size<T>(exec_q, n, nrhs, lda, ldb);
+    std::int64_t scratchpad_size;
+    sycl::event comp_event;
+    std::int64_t *ipiv = nullptr;
+    T *scratchpad = nullptr;
 
-    T *scratchpad = helper::alloc_scratchpad<T>(scratchpad_size, exec_q);
+    std::stringstream error_msg;
+    bool is_exception_caught = false;
+
+#if defined(USE_ONEMKL_INTERFACES)
+    // Use transpose::T if the LU-factorized array is passed as C-contiguous.
+    // For F-contiguous we use transpose::N.
```

Comment:
Do we need to clarify why in case of `defined(USE_ONEMKL_INTERFACES)` there is only `transpose::N` used?